vms/platformvm/txs/executor/staker_tx_verification.go

```diff
@@ -577,50 +576,6 @@ func verifyAddPermissionlessValidatorTx(
 	return nil
 }
 
-type addValidatorRules struct {
-	assetID           ids.ID
-	minValidatorStake uint64
-	maxValidatorStake uint64
-	minStakeDuration  time.Duration
-	maxStakeDuration  time.Duration
-	minDelegationFee  uint32
-}
-
-func getValidatorRules(
```

Comment:
moved to staker_tx_verification_helpers.go file. These rules won't be used by stakers verification methods only